src/context/AppProvider/AppContext.tsx

```diff
@@ -154,9 +154,12 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     })()
   }, [dispatch, wallet, supportedChains, isSnapInstalled])
 
-  useGetVotingPowerQuery(undefined, {
-    skip: !isFoxDiscountsEnabled || portfolioLoadingStatus === 'loading',
-  })
+  useEffect(() => {
+    if (!isFoxDiscountsEnabled) return
+    if (portfolioLoadingStatus === 'loading') return
+
+    dispatch(snapshotApi.endpoints.getVotingPower.initiate(undefined, { forceRefetch: true }))
```

Comment:
Refetch once when the portfolio is ready, or when `accountMetadata` (the inner dependency of `selectPortfolioLoadingStatusGranular`) changes i.e when adding a new native account, changing to a MM account that was not previously part of the account metadata etc